### PR TITLE
Allow multiple login providers

### PR DESCRIPTION
**Pull request by abutcher**

### CODE REVIEW

**Comment by detiber** [comment]
aos-ci-test

File: filter_plugins/openshift_master.py [hunk 1]
```diff
@@ -77,10 +78,10 @@ def __init__(self, api_version, idp):
         self._allow_additional = True
 
     @staticmethod
-    def validate_idp_list(idp_list):
+    def validate_idp_list(idp_list, openshift_version):
         ''' validates a list of idps '''
         login_providers = [x.name for x in idp_list if x.login]
-        if len(login_providers) > 1:
+        if len(login_providers) > 1 and LooseVersion(openshift_version) < LooseVersion('3.2'):
```
Comment (detiber) [inline filter_plugins/openshift_master.py]: This is going to need to be updated for Origin versioning as well.

Comment (abutcher) [inline filter_plugins/openshift_master.py]: Yerp!

**Comment by abutcher** [comment]
aos-ci-test

**Comment by arilivigni** [comment]
aos-ci-test

**Comment by arilivigni** [comment]
aos-ci-test

**Pull request merged**